step.yml

```diff
@@ -1,44 +1,53 @@
 title: Xcode Archive & Export for iOS
-summary: Run the Xcode archive command and then export an .ipa from the archive.
+summary: Automatically manages your code signing assets, archives and exports an .ipa in one Step.
 description: |-
-
-  The Step archives your Xcode project by running the `xcodebuild archive` command and then exports the archive into an .ipa file with the `xcodebuild -exportArchive` command. This .ipa file can be shared, installed on test devices, or uploaded to the App Store Connect.
```

Comment:
I would keep this as the first sentence of the description. I think it's still the main feature of the step, and we can mention afterwards that it can prepare code signing assets too. (Also, the default value of the code signing input is `off`)

Reply:
Thanks, we can do that :)